jib-core/src/test/java/com/google/cloud/tools/jib/frontend/CredentialRetrieverFactoryTest.java

```diff
@@ -144,7 +144,7 @@ public void testGoogleApplicationDefaultCredentials_notGoogleContainerRegistry()
     Assert.assertFalse(
         credentialRetrieverFactory.googleApplicationDefaultCredentials().retrieve().isPresent());
 
-    Mockito.verifyZeroInteractions(mockLogger);
+    Mockito.verifyNoMoreInteractions(mockLogger);
```

Comment:
Is this necessary? I think `ZeroInteractions` are stricter to ensure there was no interaction with `mockLogger`, while `NoMoreInteractions` allows interactions up to this call.

Reply:
Ah, just saw the comment about the method being deprecated.